infra/testing/server/routes/test-sw.js

```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const globby = require('globby');
+const path = require('path');
+const templateData = require('../template-data');
+
+
+const match = '/test/:packageName/sw/';
+
+async function handler(req, res) {
+  const {packageName} = req.params;
+  const testFiles = await globby(`test/${packageName}/sw/**/test-*.mjs`) || [];
```

Comment:
Any reason to introduce `globby` here? We're using `glob` elsewhere in the project. Adding extra `devDependencies` isn't a big deal, but maybe we should consider migrating elsewhere if `globby` is better.